### PR TITLE
Feat/zn 515 add GitHub actions

### PR DESCRIPTION
**Pull request by przemyslaw-bak**
https://headstart.atlassian.net/browse/ZN-515

1. Add .gitignore file.
2. Add default github action
3. Fix typos in readme file.
4. Add dependabot file
5. Remove bitbucket pipeline

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Basic dependabot.yml file with
+# minimum configuration for two package managers
+
+version: 1
```
Comment (pzmuda1) [inline .github/dependabot.yml]: Shouldn't it be version: 2? We have that in starter and its defined like that in all examples

File: .github/workflows/default.yml
Comment (pzmuda1) [inline .github/workflows/default.yml:1]: I believe we also wanted to support storybook deployment?

File: .github/dependabot.yml (hunk 1 above)
Comment (przemyslaw-bak) [reply .github/dependabot.yml]: Done.

**Review by qh4r — APPROVED** [review]

File: .github/workflows/default.yml
Comment (przemyslaw-bak) [reply .github/workflows/default.yml:1]: Storybook deployment added.

**Review by qh4r — APPROVED** [review]

**Review by pzmuda1 — APPROVED** [review]

**Pull request merged**